media/js/base/mozilla-accordion-gatrack.js

```diff
@@ -6,21 +6,21 @@
 if (typeof Mozilla.Accordion === 'function') {
     Mozilla.Accordion.GLOBAL_ONEXPAND = function(section) {
         // Google Analytics event tracking
-        gaTrack([
-            '_trackEvent',
-            location.pathname + ' Accordion Interactions',
-            'Expand',
-            section.$header.text()
-        ]);
+        window.dataLayer = window.dataLayer || [];
```

Comment:
If we do end up needing this line repeated in each JS file (see above comment), shouldn't we be able to just make this call once per file?